site/guide/configuration/customize-your-dashboard.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Customize your dashboard"
+date: last-modified
+---
+
+Personalize what you see on your ValidMind dashboard when you first log in.  Rearrange the position of widgets, and add your own widgets from reporting for quick access.
+
+:::{.callout}
+Changes are automatically saved to your account and will not affect other users.
+:::
+
+
+## Rearrange widgets
+
+1. {{< var link.login >}}
+
+2. On the Dashboard, hover over the title of a widget until the **{{< fa hand>}}** appears. 
+
+3. Click and drag to position your widget as desired.
+
+## Add report widgets
+
+Along with the default **Model Findings** and **Recent Activity** widgets, widgets from [Reports](/guide/model-validation/view-reports.qmd) can be added to your dashboard:
+
+1. {{< var link.login >}}
+
+2. In the left sidebar, click **Reports**.
+
+3. Hover over the title of the widget you'd like to place on the Dashboard until the **{{< fa ellipsis-vertical>}}** appears, then click on it.
+
+4. After you select **{{< fa plus>}} Add to Dashboard**, the widget will appear on your personal view of the dashboard.
+
+    You can then rearrange the position of the widget to your liking.[^1] 
+
+#### Remove report widgets
```

Comment:
```suggestion
## Remove report widgets
```

H4 is wrong here, I think?

Reply:
I think this is mostly a style quirk, as I like nesting some of the "related" tasks under a larger header. I agree that this could be an H2, but I will make the adjustment to have the full instructions locally.